tests/test_e2e/test_cloud_spec.py

```diff
@@ -0,0 +1,47 @@
+import ops
+import pytest
+
+import scenario
+
+
+@pytest.fixture(scope="function")
```

Comment:
What's the benefit of this being a fixture rather than just a normal top-level class?